Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.3
```

Comment:
I'm afraid we can't bump the tools version: swift-crypto supports back to 5.1. We _can_ add a new Package.swift if we have to, though it complicates matters a bit.

Is there any particular reason not to just define WIN32_LEAN_AND_MEAN in all cases?

Reply:
The best reason that I have against that is hygiene and the reminder that it is in the user's namespace (it is not underscore prefixed, so this symbol is perfectly cromulent to be used by a user in their macOS program).  I can just unconditionally define it.  We could also sprinkle it amongst the various source files as necessary under a guard.

Reply:
I think I'm happy to unconditionally define it for this target. cSettings are totally available in the 5.1 tools version, we just can't use _platform-specific_ settings.

Reply:
I can agree to that and dislike it right? :-p.  Done.